### PR TITLE
Docs: Quickstart - added documentation about the `--force` option

### PR DESCRIPTION
**Pull request by michaellee**
Thought this would be helpful in scenarios where folks want to create a new Jekyll project in an existing folder that is non-empty.

### CODE REVIEW

**Comment by envygeeks** [comment]
That's better left as a sub-note with a strong warning rather than a casual fly-by mention.

File: site/_docs/quickstart.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ For the impatient, here's how to get a boilerplate Jekyll site up and running.
 # => Now browse to http://localhost:4000
 {% endhighlight %}
 
-If you wish to install jekyll into the current directory, you can do so by
-alternatively running `jekyll new .` instead of a new directory name.
+If you wish to install jekyll into an existing directory, you can do so by
+alternatively running `jekyll new .` from within the directory instead of creating a new one. If the existing directory isn't empty, you'll also have to pass the `--force` option like so `jekyll new . --force`.
```
Comment (parkr) [inline site/_docs/quickstart.md]: Let's remove `alternatively`. So confusing to me.

**Comment by parkr** [comment]
Thanks, @michaellee! Just one comment. :point_up:

**Comment by michaellee** [comment]
Thanks @envygeeks and @parkr for the feedback. I've removed `alternatively` from the sentence and pushed up the update.

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Pull request merged**